src/index.html

```diff
@@ -9,6 +9,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo"
+        >
```

Comment:
Make good formatting of code: https://mate-academy.github.io/style-guides/htmlcss.html#good